core/src/i18n/en.yaml

```diff
@@ -964,6 +965,16 @@ instances:
   description: <0>Service Instance</0> is a provisioned instance of a service that can be consumed by an application.
   no-specific-plan-parameters:
     tooltip: The service provider did not define specific parameters for the selected plan. Refer to the documentation to learn about the required parameters, and define them as a JSON in the editor.
+  statuses:
```

Comment:
968-977: Do we need the uppercase for all words? Maybe we can use it only for the first one?

Reply:
We use the same convention in 293-295 & 801, so I decided to have it like this. They are to be uppercased anyway...